src/Data/Textile/Simulator.elm

```diff
@@ -424,43 +425,56 @@ computeSpinningImpacts db ({ inputs } as simulator) =
             )
 
 
-computeFabricImpacts : Simulator -> Simulator
-computeFabricImpacts ({ inputs, lifeCycle } as simulator) =
+computeFabricImpacts : Db -> Simulator -> Result String Simulator
+computeFabricImpacts db ({ inputs, lifeCycle } as simulator) =
     let
         fabricOutputMass =
             lifeCycle
                 |> LifeCycle.getStepProp Label.Fabric .outputMass Quantity.zero
     in
-    simulator
-        |> updateLifeCycleStep Label.Fabric
-            (\({ country } as step) ->
-                let
-                    { kwh, threadDensity, picking, impacts } =
-                        case inputs.product.fabric of
-                            Product.Knitted process ->
-                                Formula.knittingImpacts step.impacts
-                                    { elec = process.elec
-                                    , countryElecProcess = country.electricityProcess
-                                    }
-                                    step.outputMass
+    db.processes
+        |> Process.loadWellKnown
+        |> Result.map
+            (\wellKnown ->
+                simulator
+                    |> updateLifeCycleStep Label.Fabric
+                        (\({ country, knittingProcess } as step) ->
+                            let
+                                productDefaultKnittingProcess =
+                                    knittingProcess
+                                        |> Maybe.withDefault KnittingProcess.Mix
```

Comment:
We could also define a default knitting process uuid for each product in products.json, and resolve them automatically. This would allow to specify different knitting techniques for different products, which seems very interesting and powerful to me, and very simple to implement. Thoughts?

Reply:
at the moment there's already a default knitting process for each knitted product, in the `fabric` key. It's not really used here, and I was about to just get rid of the `fabric` process id and only keep `knitting` or `weaving`, as Alban confirmed that we won't need a "different default value for different products", and it's already always the same process for weaved products...

So I guess I went halfway: we don't need nor want different default values (at least for now?), but I didn't remove the `processUuid` from `fabric` either... so, what do you think, which way should I go?